src/fragments/lib/auth/flutter/signin_next_steps/20_confirm_sms_mfa.mdx

```diff
@@ -0,0 +1,40 @@
+If the next step is `confirmSignInWithSmsMfaCode`, Amplify Auth has sent the user a random code over SMS and is waiting for the user to verify that code. 
+To handle this step, your app's UI must prompt the user to enter the code. After the user enters the code, pass the value to the `confirmSignIn` API.
+
+<Callout>
+
+The result includes an `AuthCodeDeliveryDetails` member. It includes additional information about the code delivery, such as the partial phone number of 
+the SMS recipient, which can be used to prompt the user on where to look for the code.
```

Comment:
```suggestion
the SMS recipient, which can prompt the user on where to look for the code.
```

Reply:
This is not saying the same thing IMO. It requires that the developer take action with the delivery details to prompt the user.